src/index.html

```diff
@@ -17,16 +17,22 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+    >
+      <input
+        class="input input__tablet"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        data-qa="big"
+      />
+      <input
+        class="input input__mobile"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        data-qa="small"
+      />
+    </form>
```

Comment:
You should have two forms, one form for big input and one for smaller.
`data-qa="big"` attribute should be set in form tag for big input, and only `data-qa="keypress"` should be in input inside big form